lib/check.h

```diff
@@ -26,6 +26,7 @@
 #include "tokenize.h"
 #include "settings.h"
 #include "errorlogger.h"
+#include "cwe.h"
```

Comment:
I had to create another header because when I tried to expand the ErrorMessage class to include CWE ids (#include "check.h") I had some problems due to some classes not yet defined (e.g. Severity).

I don't have yet, enough knowledge of the structure of the code to place the CWE mini struct in the right file.

If you want I can try to place it in errorlogger.h, it's up to you.